moped-editor/src/components/GridTable/Filters.js

```diff
@@ -196,12 +204,12 @@ const GridTableFilters = ({
       const filtersNewState = { ...filterParameters };
 
       // Find the field we need to gather options from
-      const fieldIndex = query.config.filters.fields.findIndex(
+      const fieldIndex = filtersConfig.fields.findIndex(
```

Comment:
This is also a bigger change. One initializing a gqlabstract instance, this filter config was copied into the instance from the `PROJECT_LIST_VIEW_FILTERS_CONFIG` object and accessed that way. Now, the fields are retrieved from the original `PROJECT_LIST_VIEW_FILTERS_CONFIG` object.

Reply:
super minor, but you can use `Array.prototype.find()` instead of `findIndex()` to grab the field details object in one line.

Reply:
also, btw, this is a great change!

Reply:
thx - and good catch! I'll update that to condense it down. 🚀